src/Microsoft.Health.Dicom.Web/Startup.cs

```diff
@@ -52,15 +52,18 @@ public virtual void Configure(IApplicationBuilder app)
     }
 
     /// <summary>
-    /// Adds ApplicationInsights for telemetry and logging.
+    /// Adds ApplicationInsights for telemetry and logging. We need to migrate to Application Insights
+    /// connection strings: https://github.com/microsoft/ApplicationInsights-dotnet/issues/2560
     /// </summary>
     private void AddApplicationInsightsTelemetry(IServiceCollection services)
     {
         string instrumentationKey = Configuration["ApplicationInsights:InstrumentationKey"];
 
         if (!string.IsNullOrWhiteSpace(instrumentationKey))
         {
+#pragma warning disable CS0618 // Type or member is obsolete
             services.AddApplicationInsightsTelemetry(instrumentationKey);
```

Comment:
Instead of marking this with a pragma, can we just adopt the suggested change from https://github.com/microsoft/ApplicationInsights-dotnet/issues/2560?

```
var applicationInsightsServiceOptions = new ApplicationInsightsServiceOptions
{
    ConnectionString = $"InstrumentationKey={instrumentationKey}",
};
services.AddApplicationInsightsTelemetry(applicationInsightsServiceOptions);
```

Reply:
I only read through the issue and two of the links - should have read the third 😄 

I have updated the PR to build connection strings in both startups, and tested with AI to validate telemetry is flowing.